test-utils/utils/utils.go

```diff
@@ -160,3 +173,43 @@ func (u *Utils) CheckFinishedStatus(job string, buildNumber int) (bool, error) {
 	}
 	return result.Result == successString, nil
 }
+
+// ListFilesInBuild takes build info and list all file names with matching prefix
+// The returned file name included the complete path from bucket root
+func (u *Utils) ListFilesInBuild(job string, buildNumber int, prefix string) ([]string, error) {
+	ret, err := u.ListFilesWithPrefix(fmt.Sprintf("%v/%v/%v/%v", u.directory, job, buildNumber, prefix))
```

Comment:
This file gets it wrong all over, but for this particular function path.Join seems clearly superior?